airbyte-integrations/connectors/source-github/acceptance-test-config.yml

```diff
@@ -23,10 +23,12 @@ tests:
         comments: ["airbytehq/integration-test", "updated_at"]
         commit_comments: ["airbytehq/integration-test", "updated_at"]
         commits: ["airbytehq/integration-test", "master", "created_at"]
+        deployments: ["airbytehq/integration-test", "updated_at"]
         events: ["airbytehq/integration-test", "created_at"]
         issue_events: ["airbytehq/integration-test", "created_at"]
         issue_milestones: ["airbytehq/integration-test", "updated_at"]
         issues: ["airbytehq/integration-test", "updated_at"]
+        project_columns: ["airbytehq/integration-test", "13167124", "updated_at"]
```

Comment:
The actual `project_id` like "13167124" could be changed in the future, this is potential issue, just FYI.

Reply:
yes, but it's what we have for now
There are a lot of hard-codes in this file already:

repository name: `airbytehq/integration-test`
branch name: `master`

and my new  project_id: `13167124`